src/index.ts

```diff
@@ -152,12 +162,14 @@ export async function notarize({
   appPath,
   appleId,
   appleIdPassword,
+  ascProvider = '',
```

Comment:
This shouldn't be defaulted to an empty string, let it be undefined.  It'll fail the falsey check